tools/cloud-workstations/200_configure-hpc-toolkit.sh

```diff
@@ -16,4 +16,4 @@
 # The docker script moves this to /etc/workstation-startup.d to run
 # the hpc toolkit configuration when the workstation is brought up for
 # the first time.
-sudo -i -b -u user bash /bin/configure-hpc-toolkit.sh
+runuser user /usr/local/bin/configure-hpc-toolkit.sh &
```

Comment:
What is a point of going background here ?

Reply:
Pushing it to background is recommended on the google documentation.

The user will have control either way as code-oss should start earlier.  

No error will be presented to the user.  Their workstation would just not be configured correctly.  I'm not sure how we could present an error other than putting something in profile.d that says the system wasn't configured correctly when they start a terminal.  If they don't start a terminal, there's no easy way to alert them.